src/ocrd/resource_manager.py

```diff
@@ -104,6 +104,7 @@ def list_available(self, executable=None, dynamic=True, name=None, database=None
                 for exec_path in Path(exec_dir).glob(f'{executable}'):
                     if not exec_path.name.startswith('ocrd-'):
                         self.log.warning(f"OCR-D processor executable '{exec_path}' has no 'ocrd-' prefix")
+                        continue
```

Comment:
I thought we agreed this should **NOT** be skipped?

Reply:
https://github.com/OCR-D/core/pull/1249#discussion_r1665696308

Reply:
Sorry, you're right, reverting.